pkg/virt-handler/rest/console.go

```diff
@@ -138,20 +138,41 @@ func (t *ConsoleHandler) USBRedirHandler(request *restful.Request, response *res
 	t.stream(vmi, request, response, unixSocketDialer(vmi, unixSocketPath), stopChan)
 }
 
+func (t *ConsoleHandler) vncInUse(uid types.UID) bool {
+	// Use helper here just to handle the Mutex better
+	t.vncLock.Lock()
+	defer t.vncLock.Unlock()
+	_, inUse := t.vncStopChans[uid]
+	return inUse
+}
+
 func (t *ConsoleHandler) VNCHandler(request *restful.Request, response *restful.Response) {
 	vmi, code, err := getVMI(request, t.vmiStore)
 	if err != nil || vmi == nil {
 		log.Log.Reason(err).Error(failedRetrieveVMI)
 		response.WriteError(code, err)
 		return
 	}
+
+	uid := vmi.GetUID()
+	forceParam, err := strconv.ParseBool(request.QueryParameter("force"))
+	if err != nil {
+		log.Log.Object(vmi).Reason(err).Warningf("VNC's query parameter force")
+	}
+
+	if t.vncInUse(uid) && !forceParam {
+		err = fmt.Errorf("%s", "Ongoing VNC connection. Request denied.")
+		log.Log.Object(vmi).Reason(err).Error("It would drop the user.")
```

Comment:
```suggestion
		err = fmt.Errorf("%s", "Active VNC connection. Request denied.")
		log.Log.Object(vmi).Reason(err).Error("It would drop the active connection.")
```

Reply:
I'm also fine with `ongoing`, we should just use it consistently across all docs and messages.